wgpu/Cargo.toml

```diff
@@ -84,13 +84,13 @@ replay = ["serde", "wgc/replay"]
 angle = ["wgc/angle"]
 webgl = ["wgc"]
 
-[target.'cfg(not(target_arch = "wasm32"))'.dependencies.wgc]
+[target.'cfg(any(not(target_arch = "wasm32"), target_os = "emscripten"))'.dependencies.wgc]
```

Comment:
I don't think this configuration needs to change. Instead, it should be aligned with the current webgl path.